app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -242,10 +243,43 @@ void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex,
     }
 #endif
 
+#if defined(SPACES)
```

Comment:
This has nothing to do with the SPACES build really. We should check instead for the extension support

Reply:
Agree, will fix that. It will imply new DeviceDelegate API, since only the backend has knowledge of the extension support.